docker/docker-entrypoint.d/40-swagger-ui.sh

```diff
@@ -62,5 +62,3 @@ if [[ -n "${PORT}" ]]; then
 fi
 
 find $NGINX_ROOT -type f -regex ".*\.\(html\|js\|css\)" -exec sh -c "gzip < {} > {}.gz" \;
-
-exec nginx -g 'daemon off;'
```

Comment:
this `exec` command can be removed entirely? e.g., not needed to be moved into the `CMD [ ]`?

Reply:
Because the CMD got deleted in the Dockerfile, the one from nginx's Dockerfile is used which is surprisingly `CMD ["nginx", "-g", "daemon off;"]` together with `ENTRYPOINT ["/docker-entrypoint.sh"]`.

Entrypoint did not get overridden in this Dockerfile but it has a condition that checks for the `cmd == 'nginx'` before executing the scripts inside `docker-entrypoint.d` which makes it necessary to do a rollback here if one wants to use additional scripts.

You can have a look on my use-case [here](https://gitlab.com/biletado/apidocs/) or rather [here](https://gitlab.com/biletado/compose/-/tree/main/apidocs) if you want ;).